README.j2

```diff
@@ -34,9 +32,8 @@ This is the [Graylog Enterprise](https://hub.docker.com/r/graylog/graylog-enterp
 
 | Java Version  | Platform  | Tags  |
 |---|---|---|
+| OpenJDK 8  | `linux/amd64`, `linux/arm64` | `{{ graylog.major_version }}.{{ graylog.minor_version }}`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}-{{ graylog.release }}` |
```

Comment:
Unfortunately, we cannot provide ARM builds for Enterprise at the moment because the headless Chrome binary that we ship for the reporting feature runs on amd64 only.

Reply:
I did not think of this. I'll just remove ARM for enterprise for now.